.github/workflows/ci.yml

```diff
@@ -116,12 +116,24 @@ jobs:
   test_types:
     name: Types Test
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        package:
+          [{ name: eslint-config-eslint, directory: packages/eslint-config-eslint }]
```

Comment:
I might be missing something, but with this change, won't we end up skipping the types test in the project root? Don't we need another entry in the array?

Reply:
You're right, I fixed it.